AdaptorsDependencies/src/ivy.xml

```diff
@@ -8,10 +8,44 @@
         <conf name="test" description="Test dependencies" visibility="public" extends="runtime"/>
         <conf name="provided" description="Provided dependencies" visibility="public"/>
     </configurations>
-
+    
     <dependencies defaultconf="runtime">
+        <!-- GAFFER Dependencies -->
+        <dependency org="com.fasterxml.jackson.core" name="jackson-annotations" rev="2.9.6"/> 
+        <dependency org="com.fasterxml.jackson.core" name="jackson-core" rev="2.9.6"/>
+        <dependency org="com.fasterxml.jackson.core" name="jackson-databind" rev="2.9.6"/>
+        <dependency org="uk.gov.gchq.gaffer" name="common-util" rev="1.12.0"/>
+        <dependency org="uk.gov.gchq.gaffer" name="graph" rev="1.12.0" />
+        <dependency org="uk.gov.gchq.gaffer" name="accumulo-store" rev="1.12.0" />
+        <dependency org="uk.gov.gchq.gaffer" name="sketches-library" rev="1.12.0" />
+        <dependency org="uk.gov.gchq.gaffer" name="serialisation" rev="1.12.0" />
+        <dependency org="uk.gov.gchq.gaffer" name="type" rev="1.12.0" />
+        <dependency org="uk.gov.gchq.gaffer" name="data" rev="1.12.0" />
+        <dependency org="uk.gov.gchq.gaffer" name="operation" rev="1.12.0" />
+        <dependency org="uk.gov.gchq.gaffer" name="store" rev="1.12.0" />
+        <dependency org="uk.gov.gchq.gaffer" name="hdfs-library" rev="1.12.0" />
+        
+        <dependency org="uk.gov.gchq.koryphe" name="koryphe" rev="1.10.0">
+            <artifact name="koryphe" type="pom" ext="pom"/>
+        </dependency>
+
+        <dependency org="uk.gov.gchq.koryphe" name="core" rev="1.10.0"/>
```

Comment:
I'd just like to ask whether all of these libraries are required for the plugin to work. If so then so be it but if there is potential to exclude some parts then it would be worth it to reduce the overall memory footprint of Constellation. For instance when importing the accumulo-store, if it depends on server side libraries which also get downloaded but  will never be used (because this is a client app) then it is worth adding excludes. I am happy to help out with doing this analysis so just let me know.

Reply:
I've removed hdfs-library and accumulo-store. There may be more that can be removed. I ran the Unit Tests up quick and the queries still function.